check-load/lib/check_load.go

```diff
@@ -78,6 +80,11 @@ func run(args []string) *checkers.Checker {
 		}
 	}
 
-	msg := fmt.Sprintf("load average: %.2f, %.2f, %.2f", loadavgs[0], loadavgs[1], loadavgs[2])
+	var msg string
+	if opts.PerCPU {
+		msg = fmt.Sprintf("load average: %.2f, %.2f, %.2f", thresholdsPerCPU[0], thresholdsPerCPU[1], thresholdsPerCPU[2])
+	} else {
+		msg = fmt.Sprintf("load average: %.2f, %.2f, %.2f", loadavgs[0], loadavgs[1], loadavgs[2])
+	}
```

Comment:
This block shows values that was used to check whether is higher than the threshould, so I think this could be removed the second `if opts.PerCPU` statement.

- add `load` into the slice, eg.named `threshould`, at the end of the `for ... range loads` loop.
- print `thresholds`.

Reply:
I follow your suggestion, thanks for the review.